### DISCUSSION TITLE
i18n: pt-BR translation added, fixed typos

### DISCUSSION THREAD

**Discussion by hilam** [open-webui/open-webui#7224] [Show and tell]
Hello everyone! I'm a new contributor to Open WebUI. I'd like to discuss changes I'm proposing to the project.

### Description

I've added missing translate strings and fixed some typos in the translation for Brazilian Portuguese language, which would benefit potential Brazilian users using and managing local LLM via Open WebUI.

#### Added

* Missing translate strings

#### Fixed

* Typos
* Wrong translations
* Translations match original case

**Comment by alvarellos** [comment]
Hi Hildeberto, I have added Support for Multi Language for the Openwebui. It might be interesting for you.

Here is the discussion
[[Amber](https://github.com/amberyufangchiu)](https://github.com/open-webui/open-webui/discussions/15839)
and the PR
https://github.com/open-webui/open-webui/pull/15840

I would appreciate to have a feedback :)

Thanks !!
